### DISCUSSION TITLE
Publishing: metadata & asset validation errors

### DISCUSSION THREAD

**Discussion by tfindley15** [dandi/helpdesk#49] [Q&A]
Hi, 

I validated my NWB dataset using the dandi package in python and uploaded it. However, it is not allowing me to publish because of 1 metadata validation error and 1121 (the number of files I uploaded) asset validation errors. When I click on the errors, they both read: Metadata version 0.6.2 is not allowed. Allowed are: 0.6.0, 0.4.4, 0.5.1, and 0.5.2. 

However, I can't find anywhere in pynwb or dandi that I can specify versions and those versions do not correspond with either pynwb or dandi package versions. Can I get some guidance on how to approach this error? 

Here is the dataset I am working with: https://gui.dandiarchive.org/#/dandiset/000217/draft

**Comment by satra** [comment]
@tfindley15 - we spotted the issue last night and working to resolve it. you should not have to do anything. technically this will resolve automatically.

**Comment by tfindley15** [comment]
Thank you for your help! Will I need to re-upload the dataset once the bug
is fixed? Or will it publish automatically? Thank you again for your time!

Cheers, Reese

On Thu, Jan 20, 2022 at 8:21 AM Satrajit Ghosh ***@***.***>
wrote:

> @tfindley15 <https://github.com/tfindley15> - we spotted the issue last
> night and working to resolve it. you should not have to do anything.
> technically this will resolve automatically.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dandi/helpdesk/discussions/49#discussioncomment-2007838>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOTK2FDRUBVMMPHDZGN27I3UXAY65ANCNFSM5ML3RKRQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Teresa M Findley
Smear Lab, University of Oregon
Huestis 206B

**Reply by satra** [reply]
@tfindley15 -  we can ping here when the bug is fixed and you should be able to publish at that point. the `publish` action is left to the datanset owners.

cc: @dchiquito @waxlamp

**Comment by dchiquito** [comment]
The schema version issue has been resolved. I still see a validation error on https://gui.dandiarchive.org/#/dandiset/000217/draft, but it looks like a legitimate schema violation.

**Reply by satra** [reply]
thank you @dchiquito.

@tfindley15 - the identifier has to be url or a compact uri. the closest i see is something like this: http://www.ontobee.org/ontology/NBO?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNBO_0000323 (please use if appropriate).

**Comment by tfindley15** [comment]
Hi,

I am pretty confused at how to fill out all these fields -- the language
isn't very intuitive to me and I'm not sure what most of them are
requesting despite the documentation. Is it possible for me to meet with
someone over zoom to walk me through it? Thank you so much for all your
help!

Cheers, Reese

On Mon, Jan 24, 2022 at 7:53 AM Satrajit Ghosh ***@***.***>
wrote:

> thank you @dchiquito <https://github.com/dchiquito>.
>
> @tfindley15 <https://github.com/tfindley15> - the identifier has to be
> url or a compact uri. the closest i see is something like this:
> http://www.ontobee.org/ontology/NBO?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNBO_0000323
> (please use if appropriate).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dandi/helpdesk/discussions/49#discussioncomment-2035213>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOTK2FGI2SS77P4LR4KU5ZDUXVYWRANCNFSM5ML3RKRQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Teresa M Findley
Smear Lab, University of Oregon
Huestis 206B

**Reply by satra** [reply]
thanks @tfindley15 for the feedback. we will improve documentation for these. in the meantime, please feel free to use this meetme.so/satra to suggest some times for a meeting.
